docker/README.md

```diff
@@ -13,6 +13,8 @@ docker | 1.13.0
 ### Table of content
  * [Create lightning network cluster](#create-lightning-network-cluster)
  * [Connect to faucet lightning node](#connect-to-faucet-lightning-node)
+ * [Building standalone docker images](#building-standalone-docker-images)
+ * [Using bitcoind version](#using-bitcoind-version)
```

Comment:
nit: these titles and links no longer match the actual headings below.

Reply:
Thanks. I added the sub-headings for the `Using bitcoind version` heading.

Reply:
I've checked this heading and it matches the title and the link ?
```
[Building standalone docker images](#building-standalone-docker-images)
```

Reply:
Oh, sorry. I guess I missed that this was a heading that was already present in the document, just not in the table of contents. All good.